pythia

```diff
@@ -88,27 +89,23 @@ command_build() {
 
     # Build stuff statically on most platforms, install headers and libs in the
     # following locations, making sure they are picked up when building Python.
-    # $CPPFLAGS is another way to ensure this, but it's not as portable.
-    echo "#### Starting the build... ####"
-    execute mkdir -p $INSTALL_DIR/{include,lib}
+    # $CFLAGS/$CPPFLAGS is another way to ensure this, but it's not as portable.
+    execute mkdir -p "$INSTALL_DIR"/{include,lib}
     export LDFLAGS="-L${INSTALL_DIR}/lib/ ${LDFLAGS}"
     export PKG_CONFIG_PATH="${INSTALL_DIR}/lib/pkgconfig/:${PKG_CONFIG_PATH}"
 
-    build_dep $BUILD_LIBFFI   libffi    $LIBFFI_VERSION
-    build_dep $BUILD_ZLIB     zlib        $ZLIB_VERSION
-    build_dep $BUILD_BZIP2    bzip2      $BZIP2_VERSION
-    build_dep $BUILD_LIBEDIT  libedit  $LIBEDIT_VERSION
-    build_dep $BUILD_GMP      gmp          $GMP_VERSION
-    build_dep $BUILD_OPENSSL  openssl  $OPENSSL_VERSION
+    build_dep $BUILD_LIBFFI   libffi           $LIBFFI_VERSION
+    build_dep $BUILD_ZLIB     zlib             $ZLIB_VERSION
+    build_dep $BUILD_BZIP2    bzip2            $BZIP2_VERSION
+    build_dep $BUILD_LIBEDIT  libedit          $LIBEDIT_VERSION
+    build_dep $BUILD_GMP      gmp              $GMP_VERSION
```

Comment:
I don't think that we need GMP.... as the plan for python 3 is to only use python cryptogrphy and openssl

Reply:
That would be nice, GMP build + tests take quite some time.

Reply:
I think that you can remove GMP ... and if required we can add it later...but I hope we can simplify the build.

Reply:
To make it simple to readd GMP, I'll remove it in a dedicated PR. Issue added: https://github.com/chevah/pythia/issues/7.